docs/source/methodology.rst

```diff
@@ -0,0 +1,268 @@
+
+Example
+^^^^^^^
+
+The example below demonstrates this process with simple line survey shown below and the following parameters:
+
+.. image:: images/octree_padding_distance.png
+    :width: 800
+    :alt: paddings
+
+
+Horizontal extent
+#################
+
+    - Input:
+        - 5,600 m (survey hull)
+        - 2 * 1,000 m (padding distance)
+
+        **Total: 7,600 m**
+
+    - Number of cells:
+
+        .. math::
+
+            \frac{7,600 \;m}{25 \; m/cell} = 304 \; cells \\
+
+        **Round up -> 512 cells**
+
+    - Final dimensions:
+
+        .. math::
+
+            512\;cells \times 25 \frac{m}{cell} = 12,800\;m
+
+
+Vertical extent
+###############
+
+    - Input dimensions:
+        - 46 m (survey hull)
+        - 2*1000 m (vertical padding)
+        - 500 m (depth core)
+
+        **Total: 2,546 m**
+
+    - Number of cells
+        .. math::
+
+            \frac{2546 \; m}{25\; \frac{m}{cell}} = 102\; cells \\
+
+        **Round up -> 128 cells**
+
+    - Final dimensions:
+        .. math::
+
+            128 \; cells \times 25 \frac{m}{cell} = 3,200\;m
+
+Minimum refinement
+##################
+
+    - Input:
+        - 25 m (core cell size)
+        - 5 (minimum refinement)
+
+    **Largest cell dimension**:
+
+        .. math::
+
+            25\;m \times 2^{(5-1)} = 400 m
+
+
+The final mesh expected would be a 512 x 512 x 128 cells mesh, with an extent of 12,800 x 12,800 x 3,200 m. Note that the
+cell size is uniform and equal to the minimum level of 5 (400 m cell size), as defined in the parameters.
+
+
+.. _refinements:
+
+Refinements
+-----------
+
+Once the extent of the mesh has been defined, the user can increase the resolution (add small cells) in specific regions of the mesh
+based on a set of rules. In regions where no refinement is provided, the cell size will double in size until reaching
+the `minimum refinement <minimum_refinement>`_ level. Up to three refinement strategies can be applied to the mesh.
+For every refinement strategy, the user must specify the following parameters:
+
+- **Object**:
+    Geoh5 entity to be used for refinement. The type of the object dictates the method of refinement.
+        - **Points** -> Add concentric shells of cells around each vertices.
+
+            Uses the `refine_tree_from_points <refine_points>`_ method.
+        - **Curve** -> Add concentric cylinders of cells around each segment of the curve.
+
+            Uses the `refine_tree_from_curve <refine_curve>`_ method.
+        - **Surface** -> Refine the mesh on the faces of a triangulated surface in 3D.
+
+            Uses the `refine_tree_from_triangulation <refine_triangulation>`_ method.
+
+- **Levels**:
+    List of integers defining the number of cells requested at each octree level.
+
+    .. math::
+        [1^{st}, 2^{nd}, 3^{rd}, ...]
+
+- [Optional] **Define as horizon**:
+    If checked, the object will be used to define a horizon. The vertices of the object are used to first
```

Comment:
This must only be for surface?  If so mention it here

Reply:
Is this equivalent to the 'Refine by layers' method covered below?

Reply:
I changed my mind on naming - fixing.